BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00158
\ No newline at end of file
+1.0.25-prerelease-00177
```

Comment:
Can we bump this to build 00180?  It's needed to pick up some recent changes to support MicroBuild v2.